app/assets/stylesheets/content/_in_place_editing.sass

```diff
@@ -5,7 +5,7 @@ $inplace-edit--color--very-dark: #cacaca
 .inplace-edit
   @extend .form--field.-full-width
 
-  &.-preview
+  .-preview
```

Comment:
@0xF013 modifier-type classes should never be used standalone – but we can adjust this in another PR / @floriank